meshpy/mesh.py

```diff
@@ -808,6 +808,10 @@ def display_pyvista(
         """
 
         plotter = pv.Plotter()
+
+        plotter.enable_parallel_projection()
```

Comment:
Sorry, there was a misunderstanding, the `enable_parallel_projection` should be optional (turned off by default). And the axes can be set to on all the time.